x-pack/plugins/cases/public/components/all_cases/selector_modal/use_cases_add_to_existing_case_modal.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useCallback } from 'react';
+import { AllCasesSelectorModalProps } from '.';
+import { CasesContextStoreActionsList } from '../../cases_context/cases_context_reducer';
+import { useCasesContext } from '../../cases_context/use_cases_context';
+
+export const useCasesAddToExistingCaseModal = (props: AllCasesSelectorModalProps) => {
+  const context = useCasesContext();
+  const closeModal = useCallback(() => {
+    context.dispatch({
+      type: CasesContextStoreActionsList.CLOSE_ADD_TO_CASE_MODAL,
+    });
+  }, [context]);
```

Comment:
We only need the `dispatch` function inside these two functions, why not use only the `dispatch` function in the dependencies, instead of the whole context? We know the dispatch function never changes.

Reply:
Thanks. Fixed.